### PR TITLE
TECH-1446 Use webpack to load js

### PR DESCRIPTION
**Pull request by AKarmanov**
## JIRA

https://jira.jahia.org/browse/TECH-1446

## Description

Use webpack to load js

### CODE REVIEW

File: src/main/resources/jnt_person/html/person.jsp [hunk 1]
```diff
@@ -5,24 +5,9 @@
 <%@ taglib prefix="template" uri="http://www.jahia.org/tags/templateLib" %>
 <%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
 <template:addResources type="css" resources="person.css"/>
-<template:addResources type="javascript" resources="jquery.min.js"/>
-<template:addResources type="javascript" resources="jquery.fancybox.js"/>
 <template:addResources type="css" resources="jquery.fancybox.css"/>
```
Comment (tdraier) [inline src/main/resources/jnt_person/html/person.jsp:8]: can we get rid also of the assets dependency on css (include it in the js with style-loader ?)

File: pom.xml [hunk 2]
```diff
@@ -49,7 +49,7 @@
   <parent>
     <groupId>org.jahia.modules</groupId>
     <artifactId>jahia-modules</artifactId>
-    <version>8.0.0.0</version>
+    <version>8.2.0.0-SNAPSHOT</version>
```
Comment (tdraier) [inline pom.xml:52]: not really an issue but what is the need to update the version here ?

Comment (AKarmanov) [reply pom.xml:52]: We agreed to update versions in all modules to 8.2.0.0-SNAPSHOT

File: src/main/resources/jnt_person/html/person.jsp (hunk 1 above)
Comment (AKarmanov) [reply src/main/resources/jnt_person/html/person.jsp:8]: I can try

**Pull request closed without merging**